docker/Makefile

```diff
@@ -1,19 +1,10 @@
+COMPAT_MSG="The current version of Powergate support newer versions than Testnet network. You might be interested in using the Calibration network (make calibration-up)"
```

Comment:
Some message that gets printed if `make up` is executed. Will eventually go away when Testnet resets to latest code.